src/server/utils/redis.ts

```diff
@@ -0,0 +1,8 @@
+import * as redis from 'redis'
```

Comment:
I've been told somewhere that `ioredis` is a good package. We 'implicitly' use it in giraffe, could be worth taking a look at?

Reply:
looks very neat! nice catch

Reply:
or "catch".. idk. input?